tests/rptest/tests/mirror_maker_test.py

```diff
@@ -8,38 +8,60 @@
 # by the Apache License, Version 2.0
```

Comment:
Do you think that this commit (`tests/mirror_maker: using kafka as a source broker`) should be folded into the commit which adds `tests/rptest/tests/mirror_maker_test.py`?

Reply:
i will do it